### ISSUE TITLE
Switch to use language server

### ISSUE THREAD

**Issue by ChrisChinchilla** [ChrisChinchilla/vale-vscode#23]

**Comment by ChrisChinchilla** [comment]
@jdkato Yes I have a prototype working, but hit some issues with the language server and also needed to think about how some of the features the plugin adds on top will work, so while I may release a pre-release soon, it will be a little while yet before there's something I think people can use in production.

I'd say, if the PR isn't too much work "the old way". submit it, if it is, then leave it for now.

**Comment by jdkato** [comment]
I haven't started anything yet, so I'll hold off for now.
